src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -1,173 +1,173 @@
-  --#{$menu-toggle}--m-primary--hover--BackgroundColor: var(--#{$pf-global}--primary-color--200);
-  --#{$menu-toggle}--m-primary--focus--BackgroundColor: var(--#{$pf-global}--primary-color--200);
-  --#{$menu-toggle}--m-primary--active--BackgroundColor: var(--#{$pf-global}--primary-color--200);
-  --#{$menu-toggle}--m-primary--m-expanded--BackgroundColor: var(--#{$pf-global}--primary-color--200);
-  --#{$menu-toggle}--m-primary--m-expanded--Color: var(--#{$pf-global}--Color--light-100);
-
-  // Secondary
-  --#{$menu-toggle}--m-secondary--Color: var(--#{$pf-global}--primary-color--100);
-  --#{$menu-toggle}--m-secondary--BorderRadius: var(--#{$pf-global}--BorderRadius--sm);
-  --#{$menu-toggle}--m-secondary--BackgroundColor: transparent;
-  --#{$menu-toggle}--m-secondary--before--BorderWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$menu-toggle}--m-secondary--hover--before--BorderWidth: var(--#{$pf-global}--BorderWidth--md);
-  --#{$menu-toggle}--m-secondary--focus--before--BorderWidth: var(--#{$pf-global}--BorderWidth--md);
-  --#{$menu-toggle}--m-secondary--active--before--BorderWidth: var(--#{$pf-global}--BorderWidth--md);
-  --#{$menu-toggle}--m-secondary--before--BorderColor: var(--#{$pf-global}--primary-color--100);
-  --#{$menu-toggle}--m-secondary--hover--before--BorderColor: var(--#{$pf-global}--primary-color--100);
-  --#{$menu-toggle}--m-secondary--focus--before--BorderColor: var(--#{$pf-global}--primary-color--100);
-  --#{$menu-toggle}--m-secondary--active--before--BorderColor: var(--#{$pf-global}--primary-color--100);
-  --#{$menu-toggle}--m-secondary--m-expanded--Color: var(--#{$pf-global}--primary-color--100);
-  --#{$menu-toggle}--m-expanded__toggle--m-secondary--before--BorderColor: var(--#{$pf-global}--primary-color--100);
-  --#{$menu-toggle}--m-expanded__toggle--m-secondary--before--BorderWidth: var(--#{$pf-global}--BorderWidth--md);
-
-  // Plain
-  --#{$menu-toggle}--m-plain--Color: var(--#{$pf-global}--Color--200);
-  --#{$menu-toggle}--m-plain--PaddingRight: var(--#{$pf-global}--spacer--md);
-  --#{$menu-toggle}--m-plain--PaddingLeft: var(--#{$pf-global}--spacer--md);
-  --#{$menu-toggle}--m-plain--hover--Color: var(--#{$pf-global}--Color--100);
-  --#{$menu-toggle}--m-plain--focus--Color: var(--#{$pf-global}--Color--100);
-  --#{$menu-toggle}--m-plain--active--Color: var(--#{$pf-global}--Color--100);
-  --#{$menu-toggle}--m-plain--disabled--Color: var(--#{$pf-global}--disabled-color--200);
-  --#{$menu-toggle}--m-plain--m-expanded--Color: var(--#{$pf-global}--Color--100);
+  --#{$menu-toggle}--hover__toggle-icon--Color: var(--pf-t--global--icon--color--regular);
+
+  // * Menu toggle active
+  --#{$menu-toggle}--active--BackgroundColor: var(--pf-t--global--background--color--control--default);
+  --#{$menu-toggle}--active--BorderWidth: var(--pf-t--global--border--width--button--active);
+  --#{$menu-toggle}--active--BorderColor: var(--pf-t--global--border--color--clicked);
```

Comment:
@lboehling is there a use case for active/clicked styling of a menu toggle that would differ from its expanded state?